acsoo/addons.py

```diff
@@ -80,31 +80,28 @@ def addons_list_depends(ctx, exclude):
 
 @click.command()
 @click.option('--database')
-@click.option('--odoo-bin', 'odoo_bin', default=None)
+@click.option('--odoo-bin', 'odoo_bin', default='odoo')
 @click.option('--odoo-config', 'odoo_config',
               type=click.Path(dir_okay=False, exists=True))
 @click.option('--git-commit', 'git_commit', is_flag=True, default=False)
-@click.option('--languages', default='')
 @click.option('--git-push', 'git_push', is_flag=True, default=False)
-@click.option('--languages', default='')
+@click.option(
+    '--languages', default='',
+    help="If specified, a msgmerge between the .pot file and the "
+         "specified language file will be performed. If the PO language "
+         "file doesn't exists, it will be created. If --languages is not "
+         "specified, the merge with the .pot will be done with all existing "
+         "PO files in the i18n folder.")
 @click.option('--git-push-branch', 'git_push_branch')
-@click.option('--git-remote-url', 'git_remote_url')
+@click.option('--git-push-remote', 'git_push_remote')
```

Comment:
and we could rename these `--git-push-repository` and `--git-push-refspec` so we keep the native git terminology.